src/components/Navigation.js

```diff
@@ -1,16 +1,27 @@
 import React from 'react';
 import { NavLink } from 'react-router-dom';
-import './main.css';
+import '../css/components/main.css';
 
-const Navigation = () => {
+const Navigation = ({ hasToken }) => {
   return (
     <div className="navlinks">
-      <NavLink activeClassName="selected" className="list" to="/" exact>
-        List
-      </NavLink>
-      <NavLink activeClassName="selected" className="add" to="/add" exact>
-        Add Item
+      <NavLink activeClassName="selected" className="home" to="/" exact>
+        Home
       </NavLink>
+      {hasToken ? (
+        <NavLink activeClassName="selected" className="list" to="/list" exact>
+          List
+        </NavLink>
+      ) : (
+        ''
+      )}
+      {hasToken ? (
+        <NavLink activeClassName="selected" className="add" to="/add" exact>
+          Add Item
+        </NavLink>
+      ) : (
+        ''
```

Comment:
Wrapping these two navlinks with another div may make the nav hard to style _"tradeoffs"_
guess it will be better if we left all the navlinks at the same level to kinda ease the CSS work 🤦
Also, I feel like having the route guarding logic in another file is more reuseable and readable
Good idea though 💖